MOCC/src/configurations/NewPyTango/HealthMonitorDS.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, jsonify
+from flasgger import Swagger
+
+
+app = Flask(__name__)
+app.config['SWAGGER'] = {
+    'title': 'Health Monitor API',
+    'uiversion': 2
+}
+Swagger(app)
```

Comment:
the point of using Flasgger is to be able to call the REST functions using the Swagger UI (http://localhost:5000/apidocs/)
please add Swagger function definitions, so that this can be accomplished.